### DISCUSSION TITLE
How this theme generate <head> tag?

### DISCUSSION THREAD

**Discussion by incognito-developer** [MrGreensWorkshop/MrGreen-JekyllTheme#205] [How to] [answered]
hello, I'm not familer at jekyll theme. I tried to find <head> tag, because I want to add some functions.

But, I tried to find whole ".html" file which included <head> tag, but I failed to find it.

So, how this theme generate <head> tag?

Thank you.

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi,

The site is generated with Jekyll. For more information about Jekyll posts, which explain the format of post file names, please check the [documentation](https://jekyllrb.com/docs/posts/).

 - Use VS Code to search texts in all files. Check this [video](https://youtu.be/q8CWLpLayx0) to see how to open in VS Code. 

These links may be of assistance to you in the future.

 - [Mr. Green theme tutorials playlist](https://www.youtube.com/playlist?list=PLAymxPbYHgl-fFy5can7uZBMJtFWVcphD) on YouTube.
 - [Issues](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/issues/).
 - [Discussions](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/discussions/).
 - [jekyll docs](https://jekyllrb.com/docs/).

**Reply by incognito-developer** [reply]
Thank you! It helps to started jekyll!
